### DISCUSSION TITLE
Game Request

### DISCUSSION THREAD

**Discussion by Amidst1** [skyline-emu/skyline#336] [General]
First of all im very sorry if i am overusing this discussion page.
I wanted to request a game: Celeste
It is a pretty basic 2D game, though it's commercial, (i wonder if it already works), if not, then it's a good starting point for our dear emulator project here.
Thank you in anticipation.

**Comment by iniMerchant64** [comment]
You shouldve asked this in discord server instead. Celeste already boots and devs are working on fixing the ingame glitches. Please dont open issues to ask questions, instead next time just join the server and ask in the general channel

**Reply by Amidst1** [reply]
i knew I was doing this discussion page thing wrong, im very sorry, after all i don't have acess to discord for all of that :(

**Reply by iniMerchant64** [reply]
In that case if you want to stay up to date with the project id recommend making an account. Basicly most of the info can be found in there along with a compatibility list.

**Reply by Amidst1** [reply]
Well it's a very long story, i cannot ever come back to discord, atleast not for few years, thx for letting me know tho.
